docker/docker-compose-3/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.8"
+
+services:
+  mysql:
+    image: mysql:latest
+    container_name: mysql
+    ports:
+      - "3306:3306" 
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_PASSWORD}
+      MYSQL_ROOT_USER: ${MYSQL_USER}
+      MYSQL_DATABASE: myDB
+    volumes:
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
+    restart: always
+        
+  api:
```

Comment:
add "image" parameter so when the docker builds the image from the dockerfile it will have an informative name